HLTrigger/Configuration/python/HLTrigger_EventContent_cff.py

```diff
@@ -132,6 +132,7 @@
         'keep *_hltDeepCombinedSecondaryVertexBJetTagsInfos_*_*',
         'keep *_hltDeepCombinedSecondaryVertexBJetTagsPF_*_*',
         'keep *_hltDeepSecondaryVertexTagInfosPF_*_*',
+        'keep *_hltParticleNet*_*_*',
```

Comment:
This file is automatically generated, so these changes will be lost. Please remove them and instead file a CMSHLT JIRA request and explanation why this addition is needed, I note the wildcard in the keep adding all module instances starting with ```hltParticleNet*```

Reply:
@Martin-Grunewald thanks I didn't know that! I am going to update now the PR taking out that keep and I'll open a JIRA ticket explaining why we need that. In addition, to be more precise, we just need to add hltParticleNetDiscriminators* cause eventually we just look at the final discriminators for the efficiency measurement. Testing this now before opening a JIRA ticket.